cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md

```diff
@@ -32,12 +32,7 @@ cd skyhookdm-arrow/
 docker run -it -v $PWD:/w -w /w --privileged uccross/skyhookdm-arrow:v0.2.0 bash
 ```
 
-3. Install docker.
-```bash
-curl -o- https://get.docker.com | bash
-```
-
-3. Run the build script.
+3. Run the build script. Make sure [docker](https://docs.docker.com/get-docker/) is installed.
```

Comment:
There is no docker requirement for running the build.sh script. The docker installation should be an implicit requirement on the host if one wants to set up the development environment using docker. I suggest either omitting the "Make sure ....." or putting it at the beginning as a prerequisite for this setup method.

Reply:
Thanks. I have omitted the instruction since I think that step should be obvious.

Reply:
Looks good !